src/container/srv_target.c

```diff
@@ -1173,24 +1178,29 @@ cont_child_destroy_one(void *vin)
 			ABT_cond_wait(cont->sc_dtx_resync_cond, cont->sc_mutex);
 		ABT_mutex_unlock(cont->sc_mutex);
 
+		/* Give chance to DTX reindex ULT for exit. */
+		if (unlikely(cont->sc_dtx_reindex))
+			ABT_thread_yield();
```

Comment:
use cond?

Reply:
That is possible, but need additional data members.